sites/fast-website/src/app/components/content-placement/content-placement.ts

```diff
@@ -0,0 +1,5 @@
+import { FASTElement, attr } from "@microsoft/fast-element";
+
+export class ContentPlacement extends FASTElement {
+    @attr divider: boolean = false;
```

Comment:
```suggestion
    @attr({mode: "boolean"}) divider: boolean = false;
```

You'll want to make sure this attribute *converts* to a boolean when the attribute is added. You can do that by setting the "mode" to "boolean".

Reply:
Will do! Thank you.